### PR TITLE
Update docs and add unittests

### PR DESCRIPTION
**Pull request by Alxmrphi**
As requested. Update docs to reflect addition of new DINO models.

Added unittests in `test_transformations.py` for two of the models.

I then amended the naming conventions a bit in `extractors.py` because there was a discrepancy between model size and image patch resolution between the ViT and XCiT models. It makes it easier to follow a pattern, i.e. by specifying patch resolution with `p` prior to the number in the ViT numbers (which didn't do it already, but XCiT models did). This steps away from the naming conventions used in the source material, so I'm not sure it's the best idea but from a user perspective, I think it adds more clarity. However, might slip up someone who knows the original conventions. Basically, `ViT-S/16` -> `(dino)-vit-small-p16`. I'm totally open to amending this to be more in line with naming conventions used elsewhere, if others disagree it is a helpful change. No problem to change it back again :) 

Also, prior update had named the ResNet version of DINO (`dino-resnet50`) but I changed it to `dino-rn50` to be in line with the naming convention of the other ResNet models in `thingsvision`.

### CODE REVIEW

**Comment by Alxmrphi** [comment]
I can see some unit tests are failing. I verified everything in local python but had trouble running the unit test properly on my laptop so if it's a really obvious mistake, would appreciate a pointer as someone else will likely see this soon as I won't be able to check back in for next 24 hours.

**Comment by LukasMut** [comment]
@Alxmrphi, could you rebase and try again? I think I've found the issue with the unittests.

**Comment by LukasMut** [comment]
> As requested. Update docs to reflect addition of new DINO models.
> 
> 
> 
> Added unittests in `test_transformations.py` for two of the models.
> 
> 
> 
> I then amended the naming conventions a bit in `extractors.py` because there was a discrepancy between model size and image patch resolution between the ViT and XCiT models. It makes it easier to follow a pattern, i.e. by specifying patch resolution with `p` prior to the number in the ViT numbers (which didn't do it already, but XCiT models did). This steps away from the naming conventions used in the source material, so I'm not sure it's the best idea but from a user perspective, I think it adds more clarity. However, might slip up someone who knows the original conventions. Basically, `ViT-S/16` -> `(dino)-vit-small-p16`. I'm totally open to amending this to be more in line with naming conventions used elsewhere, if others disagree it is a helpful change. No problem to change it back again :) 
> 
> 
> 
> Also, prior update had named the ResNet version of DINO (`dino-resnet50`) but I changed it to `dino-rn50` to be in line with the naming convention of the other ResNet models in `thingsvision`. 

The naming is fine with me. I agree that this is indeed a helpful change.

**Comment by LukasMut** [comment]
> I can see some unit tests are failing. I verified everything in local python but had trouble running the unit test properly on my laptop so if it's a really obvious mistake, would appreciate a pointer as someone else will likely see this soon as I won't be able to check back in for next 24 hours.

@Alxmrphi, could you rebase and try again? I think I've found the issue with the unittests and resolved it.

**Comment by Alxmrphi** [comment]
Done! I couldn't locate any in-line comments though. I had presumed you had solved an issue and then removed them, hence needing the rebase. However, if I've blindly overlooked them please let me know and I will amend the PR to incorporate them.

**Comment by LukasMut** [comment]
> Done! I couldn't locate any in-line comments though. I had presumed you had solved an issue and then removed them, hence needing the rebase. However, if I've blindly overlooked them please let me know and I will amend the PR to incorporate them. 

Now, you should be able to see my in-line comments. For some reason, they were only visible to me.

File: tests/extractor/test_transformations.py [hunk 1]
```diff
@@ -22,5 +22,18 @@ def test_transformations_cnn(self):
...
+        )
+        transforms = extractor.get_transformations()
+        self.assertTrue(isinstance(transforms, T.Compose))
+        
+        model_name = "dino-vit-small-p8"
+        extractor, _, _ = helper.create_extractor_and_dataloader(
+            model_name, pretrained=False, source="ssl"
+        )
         transforms = extractor.get_transformations()
-        self.assertTrue(isinstance(transforms, tf.keras.Sequential))
```
[the remark this answers is not in the document]
Comment (LukasMut) [reply tests/extractor/test_transformations.py:26]: I'd resolve it myself but it seems as if I cannot edit this file (at least not from GitHub mobile) 🤷🏻‍♂️

File: tests/extractor/test_transformations.py [hunk 2]
```diff
@@ -22,5 +22,18 @@ def test_transformations_cnn(self):
         extractor, _, _ = helper.create_extractor_and_dataloader(
             model_name, pretrained=False, source="keras"
         )
+
```
[the remark this answers is not in the document]
Comment (Alxmrphi) [reply tests/extractor/test_transformations.py:26]: Done!

**Comment by Alxmrphi** [comment]
Yes, I saw the comments now. It looks like that extra line came along for the ride.

**Comment by Alxmrphi** [comment]
Argh, sorry about this. It's all so messy. It seems there was another deleted line that didn't show up, defining `transform` immediately preceding. This should fix it now.

**Review by LukasMut — APPROVED** [review]
I approve this PR. Please double-check whether `test_transformations` has now been correctly modified. I see multiple changes in this file. There were two missing lines (see my in-line comment). I think the two missing lines are now added back, but it seems as if there's another `test_transformations` file in `build/lib` which wasn't there before. @Alxmrphi Was this accidentally added?

**Pull request merged**